src/test/Events/__snapshots__/EventTypes.test.tsx.snap

```diff
@@ -399,7 +399,7 @@ Array [
             aria-label="Details"
             aria-labelledby="simple-node0 expandable-event-type-row-00"
             className="pf-c-button pf-m-plain"
-            data-ouia-component-id="OUIA-Generated-Button-plain-1"
+            data-ouia-component-id="OUIA-Generated-Button-plain-5"
```

Comment:
Maybe revert this?

Reply:
Interestingly, this was changed in https://github.com/cryostatio/cryostat-web/pull/858, but this PR reverts it back. I'm really not sure why.

Reply:
hmm all dependencies are up-to-date? like `yarn install` and all?